### PR TITLE
Update automation.markdown

### PR DESCRIPTION
**Pull request by Nick4-1**
Added the 'Delete Automations' paragraph

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/automation.markdown [hunk 1]
```diff
@@ -54,3 +54,8 @@ automation:
   trigger:
   ...
 ```
+### Delete Automations
+
+When autionmations remain visible in the Home Assistant Dashboard, even after having deleted in the YAML file, you have to delete them in the UI.
+To delete them completely, go to UI/Developer Tools and find the automation you want to delete under 'Current entities'.
+From there, select the circled i and in the new window at the bottom select 'REMOVE ENTITY'.
```
Comment (frenck) [inline source/_docs/automation.markdown]:
```suggestion

### Delete Automations

When automation remains visible in the Home Assistant Dashboard, even after having deleted in the YAML file, you have to delete them in the UI.

To delete them completely, go to UI/Developer Tools and find the automation you want to delete under 'Current entities'.

From there, select the circled `i` and in the new window at the bottom select 'REMOVE ENTITY'.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Added some small language improvements as a suggestion.

In general, I think it is missing "why" this happens, maybe consider adding that.

Furthermore, the UI->Developer tools route is something I would not recommend in general. Configuration -> Entities will help more in this. It can be searched and filtered down. The entity will have an additional indication that can be described in that case as well.

File: source/_docs/automation.markdown (hunk 1 above)
Comment (Nick4-1) [reply source/_docs/automation.markdown]: I have no clue why this happens but it happened to me after some other problems.
After searching the forum, I found similar questions but no answers.
Your improvements are fine!

Your right about the route I chose, that's what was given to me as a solution so I thought it was 'the' way to go.
Shall I change it?

Comment (frenck) [reply source/_docs/automation.markdown]: yes, please

**Review by frenck — APPROVED** [review]
Thanks, @Nick4-1 👍

**Pull request merged**